### PR TITLE
samples: usb: cdc_acm: fix device log property

### PR DESCRIPTION
**Pull request by pabigot**
The Kconfig symbol for device controller log level was repeated at the
point one would expect the symbol for the device log level.  Correct
the configuration file to provide the correct name (set to the default
level), and update the driver to use the new standard way of
associating the level with the driver.

Signed-off-by: Peter A. Bigot <pab@pabigot.com>

### CODE REVIEW

**Review by finikorg — COMMENTED** [review]
Why do you use `endpoint` word here?

**Comment by pabigot** [comment]
> Why do you use `endpoint` word here?

Because I didn't dig deep into USB terminology and thought that was the term for a device on the controller.  Reworded to be "device".

**Review by nashif — APPROVED** [review]

File: samples/subsys/usb/cdc_acm/prj.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@ CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"
 CONFIG_LOG=y
 CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
 CONFIG_USB_CDC_ACM=y
-CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
+CONFIG_USB_CDC_ACM_LOG_LEVEL_INF=y
```
Comment (finikorg) [inline samples/subsys/usb/cdc_acm/prj.conf]: Those are different logs, we have `USB_DEVICE_LOG_LEVEL` defined for other samples, I do not see the reason you remove it.

It is OK to add `USB_CDC_ACM_LOG_LEVEL_INF` for this sample, please also add the same for `cdc_acm_composite`

**Review by finikorg — CHANGES_REQUESTED** [review]

File: samples/subsys/usb/cdc_acm/prj.conf (hunk 1 above)
Comment (pabigot) [reply samples/subsys/usb/cdc_acm/prj.conf]: > Those are different logs, we have `USB_DEVICE_LOG_LEVEL` defined for other samples, I do not see the reason you remove it.

Ah.  The names were too close, didn't notice that one was for `DRIVER` and one for `DEVICE`--only that I wasn't getting the log messages I wanted.  Good call.

In that case I misunderstood the role of the existing setting, adding `CDC_ACM` would be a new thing and I don't care to go through all the other samples and add it there.  I'll drop the PR.

**Comment by pabigot** [comment]
The symbols were not actually as described in the commit message; this change is not appropriate.

**Pull request closed without merging**